models/desired_lrp.go

```diff
@@ -286,10 +286,6 @@ func (d *DesiredLRP) Copy() *DesiredLRP {
 	return &newDesired
 }
 
-func (d *DesiredLRP) CreateComponents(createdAt time.Time) (DesiredLRPSchedulingInfo, DesiredLRPRunInfo) {
-	return d.DesiredLRPSchedulingInfo(), d.DesiredLRPRunInfo(createdAt)
-}
-
```

Comment:
Why was this function removed?

Reply:
We couldn't see where this function was used and removed it in 6f0b40377470eb while we were here.